proxy/http2/Http2Stream.cc

```diff
@@ -86,6 +88,7 @@ Http2Stream::main_event_handler(int event, void *edata)
     event        = VC_EVENT_ACTIVE_TIMEOUT;
     active_event = nullptr;
   } else if (e == inactive_event) {
+    inactive_event = nullptr;
```

Comment:
This is the difference.

Reply:
I misunderstand that we should not set this nullptr because `inactive_event` is scheduled by `scheduled_every()` unlike other tracked events.

```
 864       inactive_event = this_ethread()->schedule_every(this, HRTIME_SECONDS(1));
```